api/src/envVarsSchema.ts

```diff
@@ -0,0 +1,118 @@
+import * as Joi from "joi";
+import { randomString } from "./service/hash";
+
+export const envVarsSchema = Joi.object({
+  ORGANIZATION: Joi.string()
+    .min(1)
+    .max(100)
+    .required()
+    .note(
+      "In the blockchain network, each node is represented by its organization name. This environment variable sets this organization name. It is used to create the organization stream on the blockchain and is also displayed in the frontend's top right corner.",
+    ),
+  PORT: Joi.number()
```

Comment:
Joi.number().port()

Reply:
@SamuelPull I added the port validation in multiple variables.